vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -19,20 +18,21 @@
 #include "bat/ads/internal/ad_events/ad_events.h"
 #include "bat/ads/internal/ad_server/ad_server.h"
 #include "bat/ads/internal/ad_serving/ad_notifications/ad_notification_serving.h"
+#include "bat/ads/internal/ad_serving/ad_targeting/geographic/subdivision/subdivision_targeting.h"
```

Comment:
sort order

Reply:
might be blind but looks correct to me :)